inspec/a2-iam-v2-upgrade-legacy-v1-policies/controls/base.rb

```diff
@@ -0,0 +1,322 @@
+# encoding: utf-8
+# copyright: 2018, Chef Software, Inc.
+# license: All rights reserved
+
+DEFAULT_POLICY_IDS = [
+    "administrator-access",
+    "editor-access",
+    "viewer-access",
+    "ingest-access"
+]
+
+DEFAULT_ROLE_IDS = [
+    "owner",
+    "editor",
+    "viewer",
+    "ingest",
+    "project-owner"
+]
+
+include_controls 'a2-api-integration' do
+  # stuff that changed with the changes in default profiles
```

Comment:
not sure what this means. is it supposed to be default policies?

Reply:
Not sure what that means. Looks outdated to me.